op/nodedns/nodedns.go

```diff
@@ -32,8 +32,20 @@ server:
   log-replies: yes
   log-local-actions: yes
   log-servfail: yes
+  num-threads: 4
+  incoming-num-tcp: 1000
+  outgoing-num-tcp: 1000
+  num-queries-per-thread: 4096
+  outgoing-range: 8192
   rrset-roundrobin: yes
   extended-statistics: yes
+  rrset-cache-size: 256m
```

Comment:
`rrset-cache-size` is recommended to be a double of `msg-cache-size`, but it is acceptable because it is consistent with other settings of Neco.